boards/nucleo_h745zi_q.json

```diff
@@ -0,0 +1,49 @@
+{
+  "build": {
+    "core": "stm32",
+    "cpu": "cortex-m7",
+    "extra_flags": "-DSTM32H7xx -DSTM32H745xx",
+    "f_cpu": "480000000L",
+    "mcu": "stm32h745zit6",
+    "product_line": "STM32H745xx",
+    "zephyr": {
+      "variant": "nucleo_h745zi_q_m7"
+    }
+  },
+  "connectivity": [
+    "can",
+    "ethernet"
+  ],
+  "debug": {
+    "default_tools": [
+      "stlink"
+    ],
+    "jlink_device": "STM32H745ZI",
+    "onboard_tools": [
+      "stlink"
+    ],
+    "openocd_board": "st_nucleo_h745zi",
+    "svd_path": "STM32H7x5.svd"
+  },
+  "frameworks": [
+    "arduino",
+    "cmsis",
+    "stm32cube",
+    "mbed",
```

Comment:
Looks like this board doesn't support the Arduino and mbed frameworks.
https://os.mbed.com/platforms/?q=&Target+vendor=STMicroelectronics
https://github.com/stm32duino/Arduino_Core_STM32/tree/master/variants

Reply:
Removed Arduino and mbed frameworks.